src/lib/mathlib/math/test/AlphaFilterTest.cpp

```diff
@@ -280,3 +282,51 @@ TEST(AlphaFilterTest, ConvergenceTest)
 
 	EXPECT_NEAR(last_value, -100.f, 1e-4f);
 }
+
+TEST(AlphaFilterTest, runPositiveRot3)
+{
+	// GIVEN an input of 1 in a filter with a default time constant of 9 (alpha = 0.9)
```

Comment:
```suggestion
	// GIVEN an input with sample interval 1 in a filter with a time constant of 9 (alpha = 0.1)
```
Or did I understand this wrong?

Reply:
no, you're right, I blindly copy-pasted another wrong comment...